packages/terra-functional-testing/tests/jest/commands/expect/toBeAccessible.test.js

```diff
@@ -0,0 +1,42 @@
+const runAxe = require('../../../../src/commands/axe/run');
+const toBeAccessible = require('../../../../src/commands/expect/toBeAccessible');
+
+jest.mock('../../../../src/commands/axe/run');
+
+describe('toBeAccessible', () => {
+  it('should pass if no accessibility violations are found', () => {
+    runAxe.mockImplementationOnce(() => ({
+      result: {
+        violations: [],
+      },
+    }));
+
+    const result = toBeAccessible();
+
+    expect(result.pass).toBe(true);
+  });
+
+  it('should not pass if accessibility violations are found', () => {
+    runAxe.mockImplementationOnce(() => ({
+      result: {
+        violations: ['Mock Violation 1'],
+      },
+    }));
+
+    const result = toBeAccessible();
+
+    expect(result.pass).toBe(false);
+  });
+
+  it('should return a message function that indicates the reason for the assertion failure', () => {
+    runAxe.mockImplementationOnce(() => ({
+      result: {
+        violations: [],
+      },
+    }));
+
+    const result = toBeAccessible();
+
+    expect(result.message()).toEqual('expected no accessibility violations but received []');
```

Comment:
How does this read when there is an existing violation?

Reply:
It prints out all violations in the array